controller/index.php

```diff
@@ -529,11 +530,34 @@ protected function assign_status()
 				$this->template->assign_var('ACTIVE_STATUS', $vars['NAME']);
 			}
 		}
+
+		$this->template->assign_var('SHOW_STATUS', $this->valid_type_permissions());
+	}
+
+	/**
+	 * Check whether the user has permission to filter by unapproved contributions
+	 * @return bool
+	 * @throws \Exception
+	 */
+	private function valid_type_permissions()
+	{
+		$types = \phpbb::$container->get('phpbb.titania.contribution.type.collection');
+		$types_managed = $types->find_authed('validate');
+
+		// If current type id is null, it's the index page
+		$current_category_type = $this->get_category_type();
```

Comment:
This variable is useless. The ternary in the following line calls `$this->get_category_type()` again.